src/Http/Routing/test/UnitTests/Builder/MapEndpointEndpointDataSourceBuilderExtensionsTest.cs

```diff
@@ -51,91 +51,33 @@ public void MapEndpoint_TypedPattern_BuildsEndpoint()
             var endpointBuilder1 = GetRouteEndpointBuilder(builder);
 
             Assert.Equal(requestDelegate, endpointBuilder1.RequestDelegate);
-            Assert.Equal("Display name!", endpointBuilder1.DisplayName);
+            Assert.Equal("/", endpointBuilder1.DisplayName);
             Assert.Equal("/", endpointBuilder1.RoutePattern.RawText);
         }
 
-        [Fact]
-        public void MapEndpoint_StringPatternAndMetadata_BuildsEndpoint()
-        {
-            // Arrange
-            var metadata = new object();
-            var builder = new DefaultEndpointRouteBuilder();
-            RequestDelegate requestDelegate = (d) => null;
-
-            // Act
-            var endpointBuilder = builder.Map("/", "Display name!", requestDelegate, new[] { metadata });
-
-            // Assert
-            var endpointBuilder1 = GetRouteEndpointBuilder(builder);
-            Assert.Equal(requestDelegate, endpointBuilder1.RequestDelegate);
-            Assert.Equal("Display name!", endpointBuilder1.DisplayName);
-            Assert.Equal("/", endpointBuilder1.RoutePattern.RawText);
-            Assert.Equal(metadata, Assert.Single(endpointBuilder1.Metadata));
-        }
-
-        [Fact]
-        public void MapEndpoint_TypedPatternAndMetadata_BuildsEndpoint()
-        {
-            // Arrange
-            var metadata = new object();
-            var builder = new DefaultEndpointRouteBuilder();
-            RequestDelegate requestDelegate = (d) => null;
-
-            // Act
-            var endpointBuilder = builder.Map(RoutePatternFactory.Parse("/"), "Display name!", requestDelegate, new[] { metadata });
-
-            // Assert
-            var endpointBuilder1 = GetRouteEndpointBuilder(builder);
-            Assert.Equal(requestDelegate, endpointBuilder1.RequestDelegate);
-            Assert.Equal("Display name!", endpointBuilder1.DisplayName);
-            Assert.Equal("/", endpointBuilder1.RoutePattern.RawText);
-            Assert.Equal(metadata, Assert.Single(endpointBuilder1.Metadata));
-        }
-
         [Fact]
         public void MapEndpoint_AttributesCollectedAsMetadata()
         {
             // Arrange
             var builder = new DefaultEndpointRouteBuilder();
 
             // Act
-            var endpointBuilder = builder.Map(RoutePatternFactory.Parse("/"), "Display name!", Handle);
+            var endpointBuilder = builder.Map(RoutePatternFactory.Parse("/"),  Handle);
 
             // Assert
             var endpointBuilder1 = GetRouteEndpointBuilder(builder);
-            Assert.Equal("Display name!", endpointBuilder1.DisplayName);
             Assert.Equal("/", endpointBuilder1.RoutePattern.RawText);
             Assert.Equal(2, endpointBuilder1.Metadata.Count);
             Assert.IsType<Attribute1>(endpointBuilder1.Metadata[0]);
             Assert.IsType<Attribute2>(endpointBuilder1.Metadata[1]);
         }
 
-        [Fact]
-        public void MapEndpoint_ExplicitMetadataAddedAfterAttributeMetadata()
```

Comment:
Was this test moved somewhere else? If not I think it is still worth checking the precedence.

Reply:
This is no longer possible to do since we don't accept metadata in the call to `Map`. 

There is another test in this file that tests the ordering when using the builder to add metadata.

Reply:
But what about the precendence between metadata added with the extension method vs attribute metadata?

IDK, for some reason I have always been paranoid about precendence in this area 😄

Reply:
There is a separate test that tests that.